Projects/TEAM-1/Lebid Maksym/Task301/Task301/Program.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task301
+{
+    internal class Program
+    {
+        class Item
+        {
+            public string name;
+            public float cost;
+            public int amount;
+
+            public Item(string name, float cost, int amount)
+            {
+                this.name = name;
+                this.cost = cost;
+                this.amount = amount;
+            }
+        }
+        class Market
+            {
+                public float balance = 0;
+                private List<Item> Stock = new List<Item>();
+
+                public void AddGood()
+                {
+
+                    string name;
+                    float cost;
+                    int amount;
+                    Console.WriteLine("Enter name: ");
+                    name = Console.ReadLine();
+                    Console.WriteLine("Enter cost: ");
```

Comment:
This is the logic for CREATE good, but your method is named AddGood.

So it could be great to separate this logic into two methods.